src/settings/OrderTemplates/OrderTemplatesEditor/PurchaseOrderInformationForm/handleOrganizationSelect.js

```diff
@@ -0,0 +1,71 @@
+import get from 'lodash/get';
+import isObject from 'lodash/isObject';
+
+import {
+  PO_FORM_FIELDS,
+  POL_FORM_FIELDS,
+} from '../../../../common/constants';
+import { DISCOUNT_TYPE } from '../../../../components/POLine/const';
+
+export const handleOrganizationSelect = (change, formValues) => (vendor) => {
```

Comment:
Please include a header comment explaining what this function does (it returns a handler) and what _that_ function will do (apply changes based on the vendor). As is, this function feels misnamed since it returns a handler but does not actually handle anything itself.